app/__init__.py

```diff
@@ -292,6 +298,7 @@ def setup_babel(application):
     application.babel = Babel(application)
     application.babel.localeselector(get_locale)
     application.jinja_env.add_extension('jinja2.ext.i18n')
+    application.jinja_env.cache = {}
```

Comment:
did you notice any improvement from this in the end? I wonder if it would still be effective when running behind Apache WSGI